packages/wrangler/src/logger.ts

```diff
@@ -109,13 +109,13 @@ export class Logger {
 
 	debug = (...args: unknown[]) => this.doLog("debug", args);
 	debugWithSanitization = (label: string, ...args: unknown[]) => {
-		if (getSanitizeLogs() === "false") {
-			this.doLog("debug", [label, ...args]);
-		} else {
+		if (getSanitizeLogs()) {
```

Comment:
technically that's a breaking change - clearly documenting what value is `true` is important